### PR TITLE
Lolbranch

### PR DESCRIPTION
**Pull request by sloopcool**
ya tut tvoyu mamku skinul

- ves 1600 kg
- morda ssal'naya
- jopa - kolossalnaya

tebe pizdec

### CODE REVIEW

File: mamkapage.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```
Comment (MeloncholyShiro) [inline mamkapage.html:4]: adasd

File: mamkapage.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
```
Comment (MeloncholyShiro) [inline mamkapage.html:7]: asdasd

File: mamkapage.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (MeloncholyShiro) [inline mamkapage.html:1]: asdasd

File: mamkapage.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (MeloncholyShiro) [inline mamkapage.html:5]: asdasd

**Comment by sloopcool** [comment]
A tak?

**Pull request merged**